lsm-tree.cabal

```diff
@@ -420,6 +420,18 @@ test-suite lsm-prototypes-tests
     , lsm-tree:prototypes
     , tasty
 
+-- TODO: temporary flag that overrides the operating system dependents
```

Comment:
The flag doesn't have to be temporary. It's just the use of the flag in the project file that should be temporary.